orc8r/cloud/go/services/orchestrator/servicers/builder_servicer_test.go

```diff
@@ -311,12 +340,162 @@ func TestBaseOrchestratorMconfigBuilder_Build(t *testing.T) {
 				LogLevel:       protos.LogLevel_INFO,
 				EventVerbosity: -1,
 			},
+			"state": &mconfig_protos.State{
+				//expected value differs from map due to added jitter
+				SyncInterval: 68,
+			},
+		}
+
+		actual, err := buildBaseOrchestrator(&nw, &graph, "gw1")
+		assert.NoError(t, err)
+		assert.Equal(t, expected, actual)
+	})
+	// No state config added to Network Configs
+	t.Run("no state config", func(t *testing.T) {
```

Comment:
No need to add a new test for this -- just don't include the state config in one of the earlier tests